doc_src/index.hdr.in

```diff
@@ -17,7 +17,199 @@
 
 \section introduction Introduction
 
-This is the documentation for `fish`, the friendly interactive shell. `fish` is a user friendly commandline shell intended mostly for interactive use. A shell is a program used to execute other programs. For the latest information on `fish`, please visit the <a href="https://fishshell.com/">`fish` homepage</a>.
+This is the documentation for *fish* the <b>f</b> riendly <b>i</b> nteractive <b>sh</b> ell.
+
+A shell is a commandline interpreter. It reads text input from the commandline and interpretes it as commands to operating system, see: <a href="#shell">What is a Shell</a>.
+
+So shells serve as user-interface between applications and the operating system. There are many different shells. They differ on how this interface is implemented. *fish* specializes in the following ways:
+
+- <b>Extensive UI</b>: *fish* supports the user with syntax highlighting, autosuggestions, tab completions and selections lists, that can be navigated and filtered, see: <a href="#autosuggestions">Autosuggestions</a> and <a href="#completion">Tab Completions</a>.
+
+- <b>No further configuration is needed</b>: *fish* comes preconfigured so that it will be an efficient helper on the commandline out of the box.
+
+- <b>Add new commands easily</b>: in *fish* new commands can be added on the fly. The syntax is easy to learn and there is no administrative overhead, see: <a href="#syntax-function">Functions</a>.
+
+- <b>Interactive shell</b>: *fish* focuses on commands that will be run in an interactive session: While it supports job control for external commands, its own commands share the process of the shell, that started them.
```

Comment:
Good point! I just checked:

### The README: 
"fish is a smart and user-friendly command line shell for macOS, Linux, and the rest of the family. fish includes features like syntax highlighting, autosuggest-as-you-type, and fancy tab completions that just work, with no configuration required." 

I think this is okay for a README and not in contradiction to this list.

### The Website:

- Autosuggestions:
"fish suggests commands as you type based on history and completions, just like a web browser. Watch out, Netscape Navigator 4.0!"

I don't like that reference to a browser, that is not in use anymore, as far as I know, otherwise the content is okay: could be changed to: 
"fish suggests commands as you type based on history and completions, just like a web browser. "

- Glorious VGA Color:
"fish supports 24 bit true color, the state of the art in terminal technology. Behold the monospaced rainbow."
This feature seems no longer exceptional and should be skipped altogether on the webpage: [see this issue](https://github.com/fish-shell/fish-shell/issues/1535)

- Sane Scripting:
"fish is fully scriptable, and its syntax is simple, clean, and consistent. You'll never write esac again."

I struggled understanding what fully scriptable means. So I would suggest to rewrite this as:

"fish is also a programming language that differs from other shell scripting languages as it has a syntax that is simple, clean, consistent and easy to understand. Scripting can also be done interactively."

I think this does not contradict my point of "Add new commands easily", or should I change my point in "scripting with ease"?

- Web Based configuration:
"For those lucky few with a graphical computer, you can set your colors and view functions, variables, and history all from a web page."

This part is missing in the documentation! It should be added in my opinion. Then this point should also be added to the introduction. Should I do both of these things?

- Man Page Completions:
"Other shells support programmable completions, but only fish generates them automatically by parsing your installed man pages."

Okay and partly covered in my point "Extensive UI". Since there is a link to the detail part of the documentation, I think it is okay to not mention the generating of tab completions from man pages here, but it is also okay, that the advertisement of fish on the website mentions this. I feel for beginners the tab completions would be hard to understand this earlier in the documentation, this is why I left it for the later more detailed parts of the documentation. But the advertisement website might contain features that are hard to understand for beginners.  Can you agree with me on this?

- Works Out Of The Box:
"fish will delight you with features like tab completions and syntax highlighting that just work, with nothing new to learn or configure."

Okay and covered by my point "No further configuration is needed". 

- the interactivity of the shell is missing on the webpage and could replace the "Glorious VGA Color" section. For me this interactivity is a very essential part of fish that I only discovered later on. It is definitely worth a remark on the website. 

### What to do?
Should I make any changes to the website now or leave this for later and open an issue?
Do you see anything else to do for now?